src/fparser/two/Fortran2003.py

```diff
@@ -1784,9 +1795,13 @@ def match(string):
         i = line.find('::')
         if i == -1:
             return
-        return p, Proc_Component_Attr_Spec_List(
-            repmap(line[:i].rstrip())), Proc_Decl_List(
-                repmap(line[i+2:].lstrip()))
+        attr_spec_list = Proc_Component_Attr_Spec_List(
+            repmap(line[:i].rstrip()))
+        # C449 POINTER must be present in the attribute list
```

Comment:
Please sumarise all constraints associated with R445 in the docstring and point out that the other constraints are currently not tested (see for example R451)

Reply:
Done.